modules/administration-guide/partials/proc_removing-user-data.adoc

```diff
@@ -0,0 +1,164 @@
+[subs="+quotes,macros"]
+* `pass:[CHE_SYSTEM_SUPER__PRIVILEGED__MODE = true]` 
++
+Custom Resource definitions.
++
+[subs="+quotes,macros"]
+----
+spec:
+ server:
+   customCheProperties:
+     pass:[CHE_SYSTEM_SUPER__PRIVILEGED__MODE]: 'true'
+     pass:[CHE_SYSTEM_ADMIN__NAME]: '__<admin-name>__'
+     
+----
++
+If needed, use commands below for creating an `admin` user:
++
+[subs="+quotes,macros,attributes"]
+----
+$ oc patch checluster {prod-checluster} \
+  --type merge \
+  -p '{ "spec": { "server": {"customCheProperties": pass:[{"CHE_SYSTEM_SUPER__PRIVILEGED__MODE": "true"}] } }}' \
+  -n __{prod-namespace}__
+----
++
+[subs="+quotes,macros,attributes"]
+----
+$ oc patch checluster {prod-checluster} \
+  --type merge \
+  pass:[-p '{ "spec": { "server": {"customCheProperties": {"CHE_SYSTEM_ADMIN__NAME": "__<admin-name>__"}] } }}' \
+  -n __{prod-namespace}__
+----
++
+[NOTE]
+====
+All system permissions are granted to the administrative user who is configured in the `CHE_SYSTEM_ADMIN__NAME` property (the default is `admin`). The system permissions are granted when the {prod-short} server starts. If the user is not present in the {prod-short} user database, it happens after the first user’s login.
+
+.Authorization token privileges:
+
+* `admin` - Can delete all personal data of all users
+* `user` - Can delete only the data related to the user
+====
+
+* A user or an administrator is logged in the OpenShift cluster with deployed {prod-short}.
+
+* A user ID is obtained. Get the user ID using the commands below:
+
+** For the current user:
++
+[subs="+quotes,macros,attributes"]
+----
+$ curl -X GET \
+  --header 'Authorization: Bearer __<user-token>__' \
+  'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user'
+----
+// {prod-url-secure}/api/user - vs - http(s)://{prod-host}/api/user
+ 
+** To find a user by name: 
++
+[subs="+quotes,macros,attributes"]
+----
+$ curl -X GET \
+  --header 'Authorization: Bearer __<user-token>__' \
+  'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/find?name=__<username>__'
+----
+ 
+** To find a user by email: 
++
+[subs="+quotes,macros,attributes"]
+----
+$ curl -X GET \
+  --header 'Authorization: Bearer __<user-token>__' \
+  'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/find?email=__<email>__'
+----
+
+.Example of obtaining a user ID
+
+This example uses `vparfono` as a local user name.
+
+====
+[subs="+quotes,macros,attributes"]
+----
+$ curl -X GET \
+  --header 'Authorization: Bearer __<user-token>__' \
+  'https://che-vp-che.apps.che-dev.x6e0.p1.openshiftapps.com/api/user/find?name=vparfono'
+----
+
+The user ID is at the bottom of the curl command output.
+
+----
+{
+ "name": "vparfono",
+ "links": [
+   {
+.
+.
+.
+   }
+ ],
+ "email": "vitaly.parfonov@gmail.com",
+ "id": "921b6f33-2657-407e-93a6-fb14cf2329ce"
+}
+----
+====
+
+.Procedure
+
+. Update the `customresourceCR` file to permit the removal of a user’s data from the {identity-provider} database:
```

Comment:
Is the file really called `customresourceCR`?

Reply:
Updated to:

. Update the `{prod-checluster}` `CheCluster Custom` Resource (CR) definition to permit the removal of a user’s data from the {identity-provider} database: